lightningd/jsonrpc.c

```diff
@@ -263,6 +263,8 @@ static void connection_complete_ok(struct json_connection *jcon,
 				   const char *id,
 				   const struct json_result *result)
 {
+    assert(id != NULL);
```

Comment:
Nit: Make sure the indentation matches the surrounding code. Same below.

Reply:
Agreed; I already discovered this myself, and was going to fix it.